templates/serviceComponent.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="service-tabs__wrapper">
+  <div class="service-tabs">
+    <div class="service-tabs__tab--n">
+      {{#each serviceTabs}}

```

Comment:
@jsebax Do you know if we can call a diferent hbs file here?  Looks like we need to split this in two components one serviceTabsHeader and other serviceTabsContent.  

Jordy is duplicating the content and then removing it from JS